src/index.html

```diff
@@ -1,13 +1,58 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta 
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    >
+    <link rel="preconnect" 
+          href="https://fonts.gstatic.com"
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" 
+      rel="stylesheet"
+    >
+    <title>Product cards</title>
+    <link 
+      rel="stylesheet" 
+      href="./style.css"
+    >
+  </head>
+  <body>
+    <div 
+      class="card"
+      data-qa="card"
+    >
+      <img 
+        src="./images/imac.jpeg" 
+        alt="images"
+        class="card__image"
+      >
+      <p class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      <p class="card__product-code">Product code: 195434</p>
+      <div class="card__product-rate">
+        <div class="stars">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+        <p class="card__product-review">Reviews: 5</p>
+      </div>
+      <span class="card__price-word">Price:</span>
+      <span class="card__price-number">$2,199</span>
+      <a 
+        href="index.html"
+        data-qa="hover"
+        class="card__button-buy"
+      >
+      BUY
+      </a>
```

Comment:
```suggestion
      >
        buy
      </a>
```
lets make it in uppercase via css